src/tests/ftest/launch.py

```diff
@@ -755,6 +786,14 @@ def run_tests(test_files, tag_filter, args):
             if args.clean:
                 if not clean_logs(test_file["yaml"], args):
                     return 128
+            # Set prlimit
+            set_ulimit(test_file["yaml"], args)
```

Comment:
This isn't going to work.  This will set the ulimit you are trying to change for only the ssh command session that this runs.  Once it exits and control continues on down to starting the daos servers from avocado, the limits of that shell will be long gone.

`/etc/security/limits.conf` is the mechanism that you want.  That will affect limits on every shell session on the node.

Reply:
Yes. Will revert to the old method.